Chapter1/index.rst

```diff
@@ -1,8 +1,8 @@
 Chapter 1: Understanding Climate Science and Carbon Models
-==========================================================
+===========================================================
```

Comment:
**Suggestion**
I think it would be awesome to add a sub title / section say "Common terms/words to know" as a section.

And THEN start talking about Carbon models and Climate sciences. Giving a bit of background. Because over here we're jumping from Carbon models to explaining terms and that may be confusing for the reader.